catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/common/NitfHeaderAttribute.java

```diff
@@ -359,6 +369,28 @@ private NitfHeaderAttribute(String longName, String shortName,
         ATTRIBUTES.add(this);
     }
 
+    private static String convertFormat(final FileType format) {
+        if (format == FileType.NITF_TWO_ONE || format == FileType.NITF_TWO_ZERO) {
+            return NITF;
+        } else if (format == FileType.NSIF_ONE_ZERO) {
+            return NSIF;
+        }
+
+        return NITF;
```

Comment:
@jlcsmith I was thinking this corresponds to the 'text' portion of the FileType - of which there are only 3 values: NITF, NSIF, and UNKNOWN. If you don't want UNKNOWN, I would be ok with "". You've already tested NITF and it's not recognized as a NITF, so I don't think you should return that.